horace_core/sqw/@DnDBase/replicate.m

```diff
@@ -18,11 +18,16 @@
 %           - The data is expanded along the plot axes of wref that are
 %           integration axes of win.
 %           - The annotations etc. are taken from the reference dataset.
+% Optional:
+% '-set_pix' -- if provided, wref object should be sqw objects with pixels.
+%               in this case result would be sqw object(s) with pixels set
+%               to reprouce replicated image, defined by input dnd object.
```

Comment:
```suggestion
%               to reproduce replicated image, defined by input dnd object.
```
You have said both that wref should be sqw and that input object is dnd. Something is inconsistent. Also as the current class is dndbase, checking if this is appropriate for sqw input/output